packages/docs/content/slush-wallet/dapp.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: dApp Integration
+---
+
+<Callout type="info">
+	If you are using `dapp-kit`, you do not need to install any additional packages to integrate with
+	the Slush wallet. [Read how to integrate with dapp-kit.](../dapp-kit/slush)
+</Callout>
+
+Using the Slush Wallet SDK, you can allow users to connect to the Slush wallet from your dApp. The
```

Comment:
We should make it clear that this is for web only, and that extension + native work without this package.

Reply:
Maybe even a `<Callout>`